### ISSUE TITLE
Stuff to think about

### ISSUE THREAD

**Issue by tlienart** [tlienart/TutorialsBuilder.jl#1]
### soon

* [ ] have global display settings so that all plots that are produced with the same framework look the same (e.g. pyplot) so that generally all figures look kind of the same
* [ ] simplified settings for saving figures locally/globally (e.g. so that the `savefig` with `@output` goes in the right place always not some of the time)
* [ ] button to download notebook, script etc (probably in top nav like gitbook)
* [ ] good search (lunr) probably optimized a bit, can steal stuff from documenter
* [ ] button to get different viewing modes for different readers 
* [ ] add command that shows the content of the last "cell" and truncates the number of lines showed by default so that cells that aren't shown don't show their output (less need to have `;` everywhere)

### later

* [ ] allow site to be printed (requires a play with `@media` rules and stuff

**Comment by zlatanvasovic** [comment]
> [ ] allow site to be printed (requires a play with `@media` rules and stuff

Bootstrap already has it all.

**Comment by tlienart** [comment]
Cool! Have you ever tried it?

**Comment by zlatanvasovic** [comment]
Yep. Try using Print preview from your browser on https://getbootstrap.com/.

**Comment by tlienart** [comment]
Do you know how that could be automated to get a printout of every page?

**Comment by zlatanvasovic** [comment]
Something like this, https://superuser.com/questions/592974/how-to-print-to-save-as-pdf-from-a-command-line-with-chrome-or-chromium, wrapped in a script.
